index.html

```diff
@@ -0,0 +1,254 @@
+        <article class="card">
+          <div class="img"></div>
+          <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+          </div>
+        </article>
+        <article class="card">
+          <div class="img"></div>
+          <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+          </div>
+        </article>
+      </div>
+    </section>
+    <!-- The About Article -->
+    <section>
+      <article class="about_me">
+        <div class="about_me_background">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don't hesitate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
+
+        <div class="container">
+          <img src="/assets/icon-languages.png" alt="languages" />
+          <h4>Languages</h4>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="container">
+          <img src="/assets/icon-frameworks.png" alt="frameworks" />
+          <h4>Frameworks</h4>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="container">
+          <img src="/assets/icon-skills.png" alt="icon-skills" />
+          <h4>Skills</h4>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </article>
+    </section>
+    <section class="contact_me">
+      <h4 class="contact_me_header">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h4>
+
+      <form action="https://formspree.io/f/mpznwqrl" method="post">
+        <input
+          type="text"
+          name="full_name"
+          placeholder="Full name"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          name="user_email"
+          placeholder="Email address"
+          required
+        />
+        <textarea name="comment" maxlength="500" cols="20" rows="5">
+Enter text here</textarea
```

Comment:
- [ ] Kindly notice that the `textarea` tag should have `required` and `placeholder` attributes.

Reply:
Sorry, am I going to remove the text in between the "textarea" tag if I use placeholder?

Reply:
Yea

Reply:
Alright, Thanks